### ISSUE TITLE
Add more unique offworld models and textures

### ISSUE THREAD

**Issue by dyceron** [randovania/open-samus-returns-rando#451]
Some to note:
- [ ] Artifact
- [ ] Sky Temple Key
- [ ] Red Temple Key (Should we add the color variations?)
- [ ] Translators
- [ ] Visors
- [ ] Dread Aeion Abilities
- [ ] Upgrade items?
- [ ] Boost Ball
- [ ] Updated Speed Booster model (the green arrows from older 2D games)
- [ ] Generic Cave Story
- [ ] Factorio gear (for later)
- [ ] Prime Beam Combos
- [ ] Echoes Beam Combos
- [ ] Echoes Beams

**Comment by Miepee** [comment]
> Updated Speed Booster model

Disagree for readability purposes. 


> Upgrade items?

Which ones?

**Comment by dyceron** [comment]
> > Updated Speed Booster model
> 
> Disagree for readability purposes.

Idk if I agree?


> > Upgrade items?
> 
> Which ones?

Speed and Flash

**Comment by Miepee** [comment]
A big black sphere with a big green triangle is way more recognizable, especially in situations where the item is located in red vines.Using the 2d symbol would make that way harder due to the "transparency"

**Comment by dyceron** [comment]
Hmm fair enough. Could use a new model/texture though since the current one is low res

**Comment by Miepee** [comment]
Modifying the itemsphere model to have bigger UVs sounds fine to me, sure.

**Comment by dyceron** [comment]
I dont think we need to add all these models anymore. If we need more, we can discuss and add them as necessary.
